qiskit/providers/models/backendproperties.py

```diff
@@ -109,21 +117,165 @@ class BackendProperties(BaseModel):
+                      operation: str,
+                      qubits: Union[int, Iterable[int]] = None,
+                      name: str = None) -> Tuple[Any, datetime.datetime]:
+        """
+        Return the gate property of the given operation.
+
+        Args:
+            operation: Name of the gate.
+            qubits: The qubit to find the property for.
+            name: Optionally used to specify within the heirarchy which
+                  property to return.
+
+        Returns:
+            Gate property.
+
+        Raises:
+            PulseError: If the property is not found or name is specified but qubit is not.
+        """
+        try:
+            result = self._gates[operation]
+            if qubits is not None:
+                result = result[_to_tuple(qubits)]
+                if name:
+                    result = result[name]
+            elif name:
+                raise PulseError("Provide qubits to get {n} of '{o}'.".format(n=name, o=operation))
+        except KeyError:
+            raise PulseError("Could not find the desired property for {o}.".format(o=operation))
+        return result
+
+    def gate_error(self, operation: str, qubits: Union[int, Iterable[int]]) -> float:
+        """
+        Return gate error estimates from backend properties.
+
+        Args:
+            operation: The operation for which to get the error.
+            qubits: The specific qubits for the operation.
+
+        Returns:
+            Gate error of the given operation and qubit(s).
+        """
+        return self.gate_property(operation, qubits,
+                                  'gate_error')[0]  # Throw away datetime at index 1
+
+    def gate_length(self, operation: str, qubits: Union[int, Iterable[int]]) -> float:
+        """
+        Return the duration of the gate in units of seconds.
+
+        Args:
+            operation: The operation for which to get the duration.
+            qubits: The specific qubits for the operation.
+
+        Returns:
+            Gate length of the given operation and qubit(s).
+        """
+        return self.gate_property(operation, qubits,
+                                  'gate_length')[0]  # Throw away datetime at index 1
+
+    def qubit_property(self,
+                       qubit: int,
+                       name: str = None) -> Tuple[Any, datetime.datetime]:
+        """
+        Return the property of the given qubit.
+
+        Args:
+            qubit: The property to look for.
+            name: Optionally used to specify within the heirarchy which property to return.
+
+        Returns:
+            Qubit property.
+
+        Raises:
+            PulseError: If the property is not found.
+        """
+        try:
+            result = self._qubits[qubit]
+            if name is not None:
+                result = result[name]
+        except KeyError:
+            raise PulseError("Couldn't find the desired property for {q}.".format(q=qubit))
```

Comment:
```suggestion
            raise PulseError("Couldn't find the propert{name} for qubit "
                             "{qubit}.".format(name="y '" + name + "'" if name else "ies",
                                               qubit=qubit))
```

Reply:
This will be nice for the `t1` function, for instance. I just coded it up without testing, so check that it outputs something like this:
`"Couldn't find the property 't1' for qubit 10."`
`"Couldn't find the properties for qubit 10."`